addon/templates/components/bsy-input.hbs

```diff
@@ -1,8 +1,12 @@
 <label for="{{id}}">{{label}}</label>
+{{#if icon}}
+  <i class="icon-{{icon}} icon-{{inputClass}}"></i>
```

Comment:
`icon-{{inputClass}}` feels wrong to me, this may collide with icon names, what about `{{inputClass}}-icon` for example?

Reply:
My initial idea was to create an easy API so you can define the size of the input like `input-lg` based on which we resize the icon as well. I've changed the property name to `size` for better description and used it in an example as well. It's `icon--{{size}}` now.